docs/roadmap.md

```diff
@@ -113,4 +104,86 @@ A: Because job zero is security and operational stability, we can't provide spec
 
 **Q: How can I provide feedback or ask for more information?**
 
-A: For existing features, you can directly comment on issues. For anything else, please open an issue.
\ No newline at end of file
+A: For existing features, you can directly comment on issues. For anything else, please open an issue.
+
+## Launched
```

Comment:
I like the idea to provide a comprehensive high level changelog on big feature deliveries. But, I am not sure how far do we plan to go and keep the history. Parameters, batch and idempotency are from the summer last year. From the current list it's not clear how far back these launches go (could be this year). 

We could add month and year to each section, to make it more specific, or make bigger separation per year.

Reply:
I was thinking of the current year - I'll remove those three

Reply:
Removed.